valhalla/baldr/graphconstants.h

```diff
@@ -652,7 +652,8 @@ constexpr uint8_t kDefaultFlowMask =
     kFreeFlowMask | kConstrainedFlowMask | kPredictedFlowMask | kCurrentFlowMask;
 constexpr uint32_t kFreeFlowSecondOfDay = 60 * 60 * 0;         // midnight
 constexpr uint32_t kConstrainedFlowSecondOfDay = 60 * 60 * 12; // noon
-constexpr uint32_t kInvalidSecondsOfWeek = -1;                 // invalid
+constexpr uint32_t kInvalidSecondsOfWeek =
```

Comment:
I checked, this constant is not used anywhere else where it could damage smth

Reply:
lets go back to `-1` it will still get the correct value but it will be more obvious as a sentinal value and it will continue to work even if we change the number of bits we give to that field in `timeinfo`.

Reply:
right, now I remember why I felt like I had to do this:
```
[build] /home/nilsnolde/dev/cpp/valhalla/valhalla/baldr/time_info.h: In static member function ‘static valhalla::baldr::TimeInfo valhalla::baldr::TimeInfo::invalid()’:
[build] /home/nilsnolde/dev/cpp/valhalla/valhalla/baldr/time_info.h:49:66: error: conversion from ‘uint64_t’ {aka ‘long unsigned int’} to ‘unsigned int:20’ changes value from ‘18446744073709551615’ to ‘1048575’ [-Werror=overflow]
[build]    49 |     return {false, 0, 0, kInvalidSecondsOfWeek, 0, false, nullptr};
[build]       |                                                                  ^
```
while when it's explicitly `1048575` it didn't complain with a warning (aka error).

Reply:
I guess it wasn't a problem before bcs nothing ever used that constant `kInvalidSecondsOfWeek` actually to create a `time_info` object.